build-libs.sh

```diff
@@ -0,0 +1,19 @@
+# Skip Prism mock test. If you want to enable this (SKIP_MOCK_TESTS=false), you will need to setup Prism server.
+# You can follow the error instructions on how to setup and run a server prism
+export SKIP_MOCK_TESTS=true
+
+# Code linting
+./gradlew :llama-stack-client-kotlin-core:spotlessApply
+./gradlew :llama-stack-client-kotlin-client-okhttp:spotlessApply
+./gradlew :llama-stack-client-kotlin:spotlessApply
+
+./gradlew build 
+
+# Copy jars
+mkdir -p build-jars
+BUILD_JARS_DIR=$(pwd)/build-jars
+export BUILD_JARS_DIR
+echo $BUILD_JARS_DIR
+cp -a llama-stack-client-kotlin/build/libs/. $BUILD_JARS_DIR
+cp -a llama-stack-client-kotlin-client-okhttp/build/libs/. $BUILD_JARS_DIR
+cp -a llama-stack-client-kotlin-core/build/libs/. $BUILD_JARS_DIR
```

Comment:
nit: can we have an echo to print where the consolidated jar libs are copied into (i.e. $BUILD_JARS_DIR) so the dev can quickly know after running the script?

Reply:
echo $BUILD_JARS_DIR --> this already does that, line 16